lib/pact/command/cont_command.ex

```diff
@@ -305,8 +305,8 @@ defmodule Kadena.Pact.ContCommand do
   end
 
   @spec build_signatures(sign_commands :: sign_commands(), result :: list()) :: valid_signatures()
-  defp build_signatures([], result), do: {:ok, SignaturesList.new(result)}
+  defp build_signatures([], result), do: {:ok, List.flatten(result)}
```

Comment:
@EdwinGuayacan why is this required?

Reply:
Since the Signature type is returning an empty list: [ ] (when receives a nil value), and this is being concatenated with the `++` notation, it produces a list of this type [ [ ] ] which I want to avoid.

Reply:
@EdwinGuayacan the `new/1` function must return a **`Signature` struct** or an **error tuple**. 
IMO this logic behind the `nil` value regarding the `SignCommand` module should be handled in a different way. My suggestion is in the comment above ⬆️